### DISCUSSION TITLE
Let's earn the Galaxy Brain achievement...

### DISCUSSION THREAD

**Discussion by eternitywen** [community/community#38623] [Other Feature Feedback, Questions, & Ideas] [answered]
Galaxy Brain | Bronze 🥉 --> 8 accepted answers

Galaxy Brain | Silver 🥈 --> 16 accepted answers

Galaxy Brain | Gold 🥇 --> 32 accepted answers

..........

**Comment by eternitywen** [comment]
Hi

**Comment by eternitywen** [comment]
1

**Comment by xiaowoonly** [comment, marked as answer]
Well... Here is my first answer. If you accept it then I am closer.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
